storage/innobase/srv/srv0start.cc

```diff
@@ -580,11 +584,82 @@ static void create_log_files_rename(
   strcpy(logfile0, logfilename);
 
   fil_open_log_and_system_tablespace_files();
+  fil_space_t::s_redo_space = fil_space_get(dict_sys_t::s_log_space_first_id);
 
   /* For cloned database it is normal to resize redo logs. */
   ib::info(ER_IB_MSG_1068, ulonglong{lsn});
 }
 
+/* create the log file name
+@param[in/out] logfilename  buffer for log file name
+retrun the dir name lenght */
+static size_t create_log_file_name(char *logfilename) {
```

Comment:
Please, mind we need to pass the size of `logfilename`. Previously, the `sizeof` in this function worked fine but now `sizeof logfilename` will always return 8 as `logfilename` is of type `char *`

Reply:
We need to passed it with `[[maybe_unused]]` as the size is used just in `ut_a`

Reply:
ut_a is release assert, Is'nt it will always be called, even it debug or release build?

Reply:
Sorry, that's true